packages/datagrid/src/components/DataGrid/DataGrid.component.js

```diff
@@ -310,7 +320,11 @@ export default class DataGrid extends React.Component {
 				</div>
 			);
 		} else {
-			content = <AgGridReact {...this.getAgGridConfig()} />;
+			content = (
+				<React.Suspense fallback={<Icon name="talend-table" />}>
```

Comment:
#4002 mentionned that loader didn't match the UX guidelines, is it the case now?

Reply:
you should keep the same pulse animation as the existing loader:
https://github.com/Talend/ui/blob/jmfrancois/fix/aggrid-assets-api/packages/datagrid/src/components/DataGrid/DataGrid.component.js#L318-L320

Reply:
thanks ! I have updated

Reply:
oh yes indeed, I have moved the code in a component and use it.
thanks